client/me/account/main.jsx

```diff
@@ -968,6 +970,18 @@ class Account extends Component {
 									/>
 								</FormFieldset>
 							) }
+
+						{ config.isEnabled( 'sites-as-landing-page' ) && (
+							<FormFieldset>
+								<FormLabel id="account__default_landing_page">
+									{ translate( 'Sites as landing page' ) }
+									<Badge className="account__beta-badge" type="info-blue">
+										beta
```

Comment:
We need to translate this literal:

```suggestion
										translate( 'beta' )
```